core/client/src/main/java/alluxio/client/block/stream/NettyPacketReader.java

```diff
@@ -272,11 +271,10 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) {
       }
 
       RPCProtoMessage response = (RPCProtoMessage) msg;
-      Protocol.Status status = response.getMessage().<Protocol.Response>getMessage().getStatus();
-      if (!Status.isOk(status) && !Status.isCancelled(status)) {
-        // TODO(andrew): use AlluxioStatusException for Netty APIs.
-        throw new InternalException(String.format("Failed to read block %d from %s with status %s.",
-            mId, mAddress, status.toString()));
+      try {
+        response.unwrapException();
+      } catch (CanceledException e) {
+        // TODO(andrew): Ask pei why we ignore canceled status.
```

Comment:
thanks, updated the comment. +1 for replacing `cancelled` with `cancelled` globally, I'd rather do this on master when there aren't large PRs in flight